docs/docs-content/clusters/data-center/nutanix/add-nutanix-cloud-account.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: "Add Nutanix Cloud Account"
+title: "Add Nutanix Cloud Account"
+description: "Learn how to add a Nutanix cloud account in Palette."
+hide_table_of_contents: false
+tags: ["data center", "nutanix"]
+---
+
+ Once the Private Cloud Gateway (PCG) is installed, the Nutanix cloud account must be added to Palette.
+
+## Prerequisites
+
+- A Nutanix cloud registered with Palette. For more information, refer to [Register Nutanix Cloud](register-nutanix-cloud.md). 
+
+- A PCG installed that connects the Nutanix cloud with Palette. For guidance, refer to [Install Private Cloud Gateway](install-pcg.md).
+
+
+## Add Cloud Account
+
+1. Log in to [Palette](https://console.spectrocloud.com/) as a tenant administrator.
+
+2. From the **left Main Menu**, select **Tenant Settings**. 
```

Comment:
Can be also from a specific project, no? (and it should be ideally from a specific project)

Reply:
Added: 

2. Ensure you are in the correct project scope.